integration-tests/pom.xml

```diff
@@ -217,6 +217,7 @@
         <module>validator</module>
         <module>xslt</module>
         <module>zipfile</module>
+        <module>box</module>
```

Comment:
Could you please sort the modules by running `mvn process-resources -Pformat` from the top level folder?